Common/Core/MetadataHelper.cxx

```diff
@@ -26,7 +26,10 @@ MetadataHelper::MetadataHelper()
                                               "RecoPassName",
                                               "Run",
                                               "AnchorPassName",
-                                              "AnchorProduction"};
+                                              "AnchorProduction",
+                                              "ROOTVersion",
+                                              "LPMProductionTag",
+                                              "O2Version"};
```

Comment:
Does this mean that we need to hardcode all possible meta-data keys of an AO2D here? So far, it was not possible to read the LPMProductionTag from the meta section?

Reply:
@sawenzel I think actually that is a good idea, should I try to implement it in this PR? Thanks!